gulpTasks/dev.js

```diff
@@ -50,7 +51,9 @@ compiler.plugin('done', () => {
   watchHtmlPagesOnce();
 });
 
-gulp.task('dev', ['setup'], done => {
+const dev = gulp.series(setup, startDevServer);
+
+function startDevServer(cb) {
```

Comment:
You may be able to get typing here by adding something like this:
```ts
/**
 * @param {() => void} cb
 */
function startDevServer(cb) { ... }
```

Reply:
It's not worthwhile because the function is actually called by `gulp` which passes a call-back.